site/docusaurus.config.js

```diff
@@ -48,12 +48,17 @@ const config = {
   baseUrl: BASE_URL,
   onBrokenAnchors: 'warn', // TODO(johnny): Fix broken links and make this 'throw'.
   onBrokenLinks: 'throw',
-  onBrokenMarkdownLinks: 'throw',
   favicon: 'img/favicon-2.ico',
   organizationName: 'estuary',
   projectName: 'flow',
   trailingSlash: true,
 
+  markdown: {
+    hooks: {
+      onBrokenMarkdownLinks:'throw',
+    },
+  },
```

Comment:
Having this in root has been deprecated.

https://docusaurus.io/docs/api/docusaurus-config#onBrokenMarkdownLinks